rerun_py/rerun_sdk/rerun/log/lines.py

```diff
@@ -194,11 +199,17 @@ def log_line_segments(
             # Reshape even-odd pairs into a collection of line-strips of length2
             # [[a00, a01], [a10, a11], [b00, b01], [b10, b11]]
             # -> [[[a00, a01], [a10, a11]], [[b00, b01], [b10, b11]]]
-            positions = positions.reshape([len(positions) // 2, 2, 2])
+            if positions.size != 0:
+                positions = positions.reshape([len(positions) // 2, 2, 2])
+            else:
+                positions = [positions]
             instanced["rerun.linestrip2d"] = LineStrip2DArray.from_numpy_arrays(positions)
         elif positions.ndim > 1 and positions.shape[1] == 3:
             # Same as above but for 3d points
-            positions = positions.reshape([len(positions) // 2, 2, 3])
+            if positions.size != 0:
+                positions = positions.reshape([len(positions) // 2, 2, 3])
+            else:
+                positions = [positions]
```

Comment:
So if not position.size is not zero, then `positions = [positions]`. isn't this more clearly expressed as `positions = [[]]`?